src/pat/calendar/calendar.js

```diff
@@ -57,8 +57,6 @@ parser.addArgument("event-color", "blue");
 
 parser.addArgument("url", null);
 parser.addArgument("event-sources", [], undefined, true);
-parser.addArgument("event-sources-classes", [], undefined, true);
-parser.addArgument("event-sources-active", [], undefined, true);
```

Comment:
I removed this as this is a too complicated way for an early idea on how to configure multiple calendars. Removed that and simplified the code below a bit.